docs/index.md

```diff
@@ -0,0 +1,42 @@
+---
+page_title: "Provider: Mackerel"
+subcategory: ""
+description: |-
+  The terraform provider for mackerel.io
+---
+
+# Mackerel Provider
+
+The Mackerel provider provides resources to interact with a Mackerel API.
+
+## Example Usage
+
+Terraform 0.14 and later:
+
+```
+terraform {
+  required_providers {
+    mackerel = {
+      source  = "Mackerel/mackerel"
+      version = "~> 0.0.1"
+    }
+  }
+}
+
```

Comment:
[Optional] missing provider configuration.

```
variable "mackerel_api_key" {      
}

provider "mackerel" {
  api_key = var.mackerel_api_key 
}
```

Reply:
thanks!
I added provider configuration to index.md. 
https://github.com/mackerelio-labs/terraform-provider-mackerel/pull/42/commits/da6e1ff12e79c7ac9a9fa962e716f78f7959fdb6

Because I think it's important information, I also add the description to README.
https://github.com/mackerelio-labs/terraform-provider-mackerel/pull/42/commits/61c2c1fdacbbba69cecfcca3c6744129d151f1c4